src/settings.ts

```diff
@@ -64,10 +66,37 @@ export class SettingTab extends PluginSettingTab {
                 .setValue(this.plugin.settings.recreateTree)
                 .onChange(async (value) => {
                     this.plugin.settings.recreateTree = value;
+                    this.display();
                     await this.plugin.saveSettings();
                 }
             ));
         
+        if (this.plugin.settings.recreateTree) {
+            new Setting(containerEl)
+                .setName('Remove Folders From Path')
+                .setDesc('Removes the specified folders from the output path, if present. Separate folders by using a comma or a new line. Names are case insensitive.')
+                .addTextArea(text => 
+                    text
+                        .setPlaceholder('00. ARCHIVES')
+                        .setValue(this.plugin.settings.removePath.join(' ,'))
```

Comment:
One more comment I apparently didn't submit, sorry!
Could we swap the comma and space, and make the placeholder use names that help explain the feature?

```suggestion
                        .setPlaceholder('RemoveThisFolder, AndThis')
                        .setValue(this.plugin.settings.removePath.join(', '))
```